10/buster-slim/Dockerfile

```diff
@@ -51,6 +59,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
 ENV YARN_VERSION 1.21.1
 
 RUN set -ex \
+  && savedAptMark="$(apt-mark showmanual)"
```

Comment:
Missing the trialling `\` here and the other occurances
```suggestion
  && savedAptMark="$(apt-mark showmanual)" \
```

Reply:
woops, fixed. thanks!